src/Nethermind/Nethermind.Xdc/Errors/Errors.cs

```diff
@@ -0,0 +1,119 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nethermind.Xdc.Errors;
+public static class CommonErrors
+{
+    // Various error messages to mark blocks invalid. 
+    // These are private in Go, but in C# we typically use public to limit exposure within the assembly.
+    public static readonly Exception ErrUnknownBlock =
+        new Exception("unknown block");
```

Comment:
Don't use/throw base Exceptions!

you are in `CommonErrors` class, I think you can skip `Err` prefix